### DISCUSSION TITLE
Inconsistent Licensing between master and latest release

### DISCUSSION THREAD

**Discussion by DennisClark** [rustdesk/rustdesk#762] [General]
Hi rustdesk team, 
The license at https://github.com/rustdesk/rustdesk/blob/master/LICENCE 
is AGPL 3.0 
but the license at https://github.com/rustdesk/rustdesk/blob/refs/tags/1.1.9/LICENSE 
is GPL 3.0 

You may want to reconcile these.

**Comment by rustdesk** [comment]
AGPL is the final one.
